### ISSUE TITLE
Uninitialized constant Haml::Util

### ISSUE THREAD

**Issue by vovchynniko** [k0kubun/hamlit#125]
Version: 2.8.9

When rendering a Rails view I'm receiving an ActionView::Template::Error (uninitialized constant Haml::Util):

```
.../.rbenv/versions/2.3.6/lib/ruby/gems/2.3.0/gems/hamlit-2.8.9/lib/hamlit/parser.rb:29:in `call'`
```

**Comment by k0kubun** [comment]
oh wow, I'll check and fix it

**Comment by k0kubun** [comment]
I reproduced the problem locally and it was fixed in v2.8.10. Thanks to report quickly.
